packages/e2e/src/chain.test.ts

```diff
@@ -42,6 +42,13 @@ describe('chain rpc', () => {
     await check(api.rpc.chain.getBlock(hash0)).toMatchSnapshot()
     await check(api.rpc.chain.getBlock(hash1000)).toMatchSnapshot()
 
+    expect(await api.rpc('archive_unstable_body', hash1000)).toEqual(
+      expect.arrayContaining([
+        '0x280401000be3da72cc7d01',
```

Comment:
it's most probably not ok to hardcode these. How can I access the lower parts of a snapshot?

Reply:
It's fine. Block 100 isn't going to change

Reply:
any reason to not use match snapshot of the whole body?

Reply:
the thing is that `archive_unstable_body` only responds with the extrinsics array. However `toMatchSnapshot` is for the `body`, the `header` etc. So if I could have selected juste the `extrinsics` from this snaptshot, it would have been ideal.

Reply:
thanks, I have indeed no idea, and was definitely looking for help. I'll check it out in the morning.

Reply:
Thanks for the guidance, it's now using a snapshot